pkgs/development/interpreters/janet/default.nix

```diff
@@ -29,7 +25,7 @@ stdenv.mkDerivation rec {
   doInstallCheck = true;
 
   installCheckPhase = ''
-    $out/bin/janet --help
+    $out/bin/janet -e '(+ 1 2 3)'
```

Comment:
`--help` is treated as an error and prints out `unknown flag --help` in hydra. Not sure how that passed in previous builds since it fails the build on my system.